### ISSUE TITLE
[Adaptive Cards-FoodOrder.Json]: Position for "Rare", "Medium-rare" and "Well-done" radio buttons is not defined under Steak button on 'FoodOrder.Json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9242]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the Adaptive Card Visualizer application.
3. Navigate to FoodOrder.Json button by using Tab key and activate it.
4. FoodOrder adaptive card will be appeared on right sight of the pane.
5. Navigate to FoodOrder card by using Tab key.
6. Navigate to "Steak" button and activate it.
7. Navigate to "Rare", "Medium-rare" and "Well-done" radio buttons and observe the screen reader narration.

**Actual Result:**
Position for "Rare", "Medium-rare" and "Well-done" radio buttons is not defined under Steak button on 'FoodOrder.Json' card. Screen Reader is only announcing 'Rare radio button not selected'.

**Observation:** 
Same issue replicates with NVDA.

**Expected Result:**
Position should be defined for "Rare", "Medium-rare" and "Well-done" radio buttons under Steak button. Screen reader should announce as, "Rare radio button 1 of 3 not selected".

**User Impact:**
User depending on screen reader might get affected if screen reader does not announce the position of radio buttons and then users will not be able to get the information about the count of controls which might impact the efficiency of the user

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/f303ae3b-e68b-4519-8e2f-42bf883ef4ef

![Image](https://github.com/user-attachments/assets/7645b508-1d95-4cf4-97ef-feafe0747fd2)

### Card JSON

```JSON
N/A
```

**Comment by Rishikesh6521** [comment]
Original Bug Id: #7623
